### DISCUSSION TITLE
What's the meaning of value 'airTemp'

### DISCUSSION THREAD

**Discussion by cr0ste** [Hacksore/bluelinky#232] [Q&A]
Hi, 

what's the meaning of this value? Is it the temperatur inside car, outside car, air control temperature? 

And most important, how can I convert it to Celcius? 
I'm getting value '02H' which doesn't give me any clue about the format and unit.

**Comment by PierreLevres** [comment]
It is the setpoint for the airconditioning. See the interface code for the algorithm. 32 steps of half a a degree celcius starting from 14 degrees.

**Reply by cr0ste** [reply]
Ah ok, I think I found it in: 
https://github.com/Hacksore/bluelinky/blob/d085c5eccdd5d3edda29451796b1bb37bd4e35b5/src/util.ts#L45

Seems to start at 14 °C for Europe. This would mean that my current value of 02H is transformed into 15 °C. 
In the official Kia app my range is between 17 and 27 °C. 
Hm, guess I need to play around a bit to figure out if this is true. I wouldn't be surprised if the range is different for specific brands and models (Kia Niro EV in my case).

**Reply by cr0ste** [reply]
I can now confirm that start of range for Europe is at 14°C. 
After activation of climatisation via the official app with 21°C, I got the temp code "0EH", which means 14 steps, which means 7°C.

**Reply by PierreLevres** [reply]
Good. So now you can enter a warm car in the morning!
